apps/dashboard/app/controllers/apps_controller.rb

```diff
@@ -3,8 +3,9 @@
 class AppsController < ApplicationController
 
   def index
-    @core_app_groups = OodAppGroup.select(titles: %w(Files Jobs Clusters), groups: sys_app_groups)
-    @app_groups = sys_app_groups.select {|g| ! %w(Files Jobs Clusters).include?(g.title) }
+    @sys_apps = sys_app_groups
+    @dev_apps = OodAppGroup.groups_for(apps: dev_apps)
+    @usr_apps = OodAppGroup.groups_for(apps: usr_apps)
```

Comment:
Should these be `nav_*_groups as well?`

Reply:
I guess so yeah. We don't want users to star apps as their favorites here that will be omitted from the main page.

The place where we would use dev_apps and usr_apps would be in the developer views - there is where you will want to see all app dirs, even those missing manifests.

Reply:
Done.